identity-account/src/stronghold/error.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use iota_stronghold::procedures::ProcedureError;
+
+pub type IotaStrongholdResult<T> = Result<T, StrongholdError>;
+
+#[derive(Debug, thiserror::Error, strum::IntoStaticStr)]
+pub enum StrongholdError {
+  /// Caused by errors from the [iota_stronghold] crate.
```

Comment:
```suggestion
/// Caused by errors from the [`iota_stronghold`] crate.
#[derive(Debug, thiserror::Error, strum::IntoStaticStr)]
pub enum StrongholdError {
```

Pull the doc comment out and add backticks.